tensorizer/serialization.py

```diff
@@ -170,6 +189,160 @@ class TensorHash:
     hash: bytes
 
 
+class _TensorPath(tuple):
+    def serialized_(self) -> bytes:
+        if self.is_str_:
+            return self[0].encode("utf-8")
+        else:
+            # application/json-seq format
+            return b"\x1e" + json.dumps(
+                self, indent=None, ensure_ascii=True, separators=(",", ":")
+            ).encode("ascii")
```

Comment:
Why are we `ascii` encoding the JSON here?

Reply:
It needs to be converted from `str` to `bytes`, so it needs to be encoded to something, and as JSON here it's already ASCII (all non-ASCII Unicode characters in json-seq strings are backslash-escaped), so it's the most correct option.